integration-tests/src/test/java/oracle/kubernetes/operator/utils/DbUtils.java

```diff
@@ -17,14 +17,17 @@
   public static final String DEFAULT_RCU_SYS_USERNAME = "sys";
   public static final String DEFAULT_RCU_SYS_PASSWORD = "Oradoc_db1";
   public static final String DEFAULT_RCU_NAMESPACE = "rcu";
+  public static final String DEFAULT_ImagePullSecret = "docker-store";
+  public static final String DEFAULT_ImagePullPolicy = "Always";
+  private static int count = 0;
   private static final Logger logger = Logger.getLogger("OperatorIT", "OperatorIT");
 
   /**
-   * create oracle db pod in the k8s cluster.
+   * Create oracle db pod in the k8s cluster.
```

Comment:
Oracle DB - be consistent, and Oracle has to have a capital O

Reply:
Done